.github/workflows/testing.yml

```diff
@@ -29,5 +29,5 @@ jobs:
           poetry run python manage.py collectstatic
           poetry run python manage.py test
 
-      - name: Check with Pylint
-        run: modules=(*/__init__.py) && poetry run pylint -j0 --load-plugins=pylint_django --django-settings-module=go3.settings ${modules[@]%/*}
```

Comment:
Is it your recommendation to stop using pylint, or was this accidental?

Reply:
That was inadvertent and this PR can just stay closed until https://github.com/Gig-o-Matic/GO3/pull/220 is reviewed for clarity's sake.... 😄